wildvvad/utils/utils.py

```diff
@@ -0,0 +1,59 @@
+# import the necessary packages
+from collections import OrderedDict
+import numpy as np
+
+# define a dictionary that maps the indexes of the facial
+# landmarks to specific face regions
+
+# For dlib’s 68-point facial landmark detector:
+FACIAL_LANDMARKS_68 = OrderedDict([
```

Comment:
isn't this already defined somewhere else? there should not exist two instances of this

Reply:
Resolved